lib/oelite/log.py

```diff
@@ -1,22 +0,0 @@
-import logging
```

Comment:
I like the idea of an oelite.log module, and I've done some thinking about how to implement it. But it won't be implemented in terms of the python stdlib logging module, since it's way too hard to figure out what happens inside that. Instead, since porting would have to happen gradually and we'd have to stay compatible with bare print statements for a long while, it would be built from scratch, writing to sys.stdout and/or sys.stderr, be made "context-aware" (switching file descriptors to a task log file and back to the terminal would be handled in oelite.log module); that way one could for example easily keep an fd for the terminal open and make oelite.log.<configurable bad enough level> made in "task" context also be echoed immediately to the terminal.

The initial implementation would probably just make all oelite.log.foo aliases for print (+ the fd handling), and then one can gradually change the codebase over.

So I agree with kimrhh, it's easier to start from scratch when we get around to doing something about this, which it seems we all want to see happening.